proxy/proxy/rhnAuthCacheClient.py

```diff
@@ -18,7 +18,9 @@
 # in this software or its documentation.
 #-------------------------------------------------------------------------------
 
-## language imports
+# Module new has been removed in future Python versions. This needs to be adapted.
+import new # pylint: disable=import-error
```

Comment:
@sbluhm 
This line causing error 500 on the Proxy Server as there is no such module.

Reply:
This has been moved upwards from line 170.